gen/pb-ruby/lib/sigstore_protobuf_specs/version.rb

```diff
@@ -16,6 +16,6 @@
 
 module Dev
   module Sigstore
-    VERSION = '0.3.3'
```

Comment:
Should we do 0.3.4? We have typically reserved 0.x bumps for breaking changes. 

We should also start looking into cutting a 1.x release after this, but that’s out of scope for now.